### ISSUE TITLE
BDV window popup when changing to molecule with different metadata record

### ISSUE THREAD

**Issue by mjosch** [duderstadt-lab/mars-fx#29]
Dear Karl,

I noticed a minor bug in mars-fx. 
Once you show the BDV and close it again, the window does not stay permanently closed anymore. Although it works fine when browsing through molecules with identical metadata record, the BDV window always pops up again in front when switching between molecules with different metadata records.
I assume manually closing the BDV window needs to trigger an event taking care of that?

**Comment by karlduderstadt** [comment]
Hi mjosch,

Thanks a lot for reporting this issue. Makes sense this would happen. When a molecule with a new metadata record is loaded the MarsBdvFrame state is completely reset. This involved hiding and and showing the frame and completely removing the panel with the video.

**Comment by karlduderstadt** [comment]
I just tested and this is no longer an issue. Now if there is not frame open, it doesn't try to update and doesn't open a new frame. There have been substantial updates to the way BDV frames and loaded and the display options, setup. This was fixed through there changes. I can't pinpoint the commit, beyond the one above. This is not quite on the update site yet but hopefully later today or this week.
